ecommerce/certprep/static/sass/partials/views/_receipt.scss

```diff
@@ -0,0 +1,487 @@
+.receipt {
```

Comment:
Are you sure you need al these definitions? @sergivalero20

Reply:
Done!
I removed unnecessary rules.